lldb/source/Host/common/Socket.cpp

```diff
@@ -87,8 +87,8 @@ llvm::Error Socket::Initialize() {
   if (err == 0) {
     if (wsaData.wVersion < wVersion) {
       WSACleanup();
-      return llvm::make_error<llvm::StringError>(
-          "WSASock version is not expected.", llvm::inconvertibleErrorCode());
+      return llvm::createStringError("WSASock version is not expected.",
+                                     llvm::inconvertibleErrorCode());
```

Comment:
This doesn't need the `inconvertibleErrorCode` anymore, right?

Reply:
Yes, that's an oversight. I also didn't try to exhaustively convert everything, just give a sample of the kinds of expected improvements.